stable/neo4j/templates/readreplicas-hpa.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.readReplica.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "neo4j.name" . }}
```

Comment:
please add the standard labels here as well

Reply:
done

Reply:
@cpanato